c_emulator/riscv_callbacks_if.h

```diff
@@ -12,82 +18,86 @@ class callbacks_if {
 
   // Callback invoked before each step
   virtual void pre_step_callback([[maybe_unused]] hart::Model &model,
```

Comment:
Now that the implementation is in a separate file we can drop all the `[[maybe_unused]]` which should make the formatting a bit less ugly.